.github/workflows/ci-workflow.yml

```diff
@@ -25,9 +25,8 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install black flake8 mypy
-        pip install types-PyYAML types-python-dateutil types-paramiko types-retry
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f testing_requirements.txt ]; then pip install -r testing_requirements.txt; fi
```

Comment:
👍 Could you please also reflect this change in https://github.com/martomi/chiadog/blob/main/CONTRIBUTING.md

Reply:
Also does it make sense to add some CI test that prevents significant regressions on coverage?

Reply:
There's now some coverage checking, sample report: https://github.com/martomi/chiadog/actions/runs/4212644589/attempts/1#summary-11439794288

I set the thresholds somewhat arbitrarily, happy to adjust. The total coverage is low due to the live testing of notifiers. They could be made hermetic with `vcrpy` but that's a bit of an undertaking!

Reply:
Nice! thresholds look good for a start!